Python/pythonrun.c

```diff
@@ -1535,12 +1063,42 @@ _PyErr_Display(PyObject *file, PyObject *unused, PyObject *value, PyObject *tb)
         }
     }
 
+    if (!value) {
+        goto fallback;
+    }
+
+    // Try first with the traceback.
+    int unhandled_keyboard_interrupt = _PyRuntime.signals.unhandled_keyboard_interrupt;
+    PyObject *traceback_module = PyImport_ImportModule("traceback");
+
+    if (traceback_module == NULL) {
+        goto fallback;
+    }
+
+    PyObject *print_tb_func = PyObject_GetAttrString(traceback_module, "_print_exception_bltin");
```

Comment:
There is actually a `print_tb` function in the traceback module. Maybe rename this variable to avoid confusion.